stellar-common/src/asset.rs

```diff
@@ -28,6 +28,15 @@ fn map_issue_asset(block: Block) -> Result<Assets, substreams::errors::Error> {
                 // We listen on the ChangeTrust operation. This will tell us if a new asset was created
                 // or if an asset was updated (the trustline was updated).
                 // The sink running this piece of code, should ignore any asset that have already been created.
+                //
+                // In the above comment:
+                //
+                // > This will tell us if a new asset was created
+                //
+                // is not true. Assets are created (minted, come into being)
+                // only when the issuer actually makes a payment with that
+                // asset. Though the ChangeTrust can be a hint that it may be
+                // soon to exist, it does not mean it will.
```

Comment:
ok good point, what would you recommend we do here?

Reply:
Well you don't really care about the asset until it's being used, right? So if you track just the payments you'll see its first usage the same way you'd see its second, Nth usage. I'm assuming the distinction of "first ever payment" doesn't matter to you?